genienlp/run_generation.py

```diff
@@ -279,37 +441,75 @@ def parse_argv(parser):
     parser.add_argument("--input_file", type=str, help="The file from which we read prompts.")
     parser.add_argument('--input_column', type=int, required=True,
                         help='The column in the input file which contains the input sentences.')
+    parser.add_argument('--prompt_column', type=int, default=None,
+                        help='The column in the input file which contains the text we should start generation from.')
+    parser.add_argument('--gold_column', type=int, default=None,
+                        help='The column in the input file which contains the gold sentences. Defaults to --input_column if no gold is available.')
+    parser.add_argument('--thingtalk_column', type=int, default=None,
+                        help='The column in the input file which contains the ThingTalk program.')
```

Comment:
Why do you need the ThingTalk program in this script? I thought we do the filtering as a separate pass?

Reply:
It is necessary to identify parameter values and capitalize them for cased models.

Reply:
Wouldn't that be a separate preprocessing? It has nothing to do with generation...